### PR TITLE
Added talk command

### PR DESCRIPTION
**Pull request by JOPHESTUS**

### CODE REVIEW

**Comment by lol768** [comment]
:/

**Comment by DSH105** [comment]
@lol768 ;o

File: src/main/java/com/dsh105/nexus/command/module/admin/TalkCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class TalkCommand extends CommandModule {
+
+    @Override
+    public boolean onCommand(CommandPerformEvent event) {
+        StringBuilder b = new StringBuilder();
+        for (int i = 1; i < event.getArgs().length; i++) {
+            if (i != 1)
+                b.append(" ");
+            b.append(event.getArgs()[i]);
+        }
```
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/command/module/admin/TalkCommand.java:39]: `com.dsh105.nexus.util.StringUtil#combineSplit(int startIndex, String[] string, String separator)`

**Comment by lol768** [comment]
Should this be free to use?

**Comment by lol768** [comment]
Error — The Travis CI build could not complete due to an error · Details

**Comment by lol768** [comment]
```
[INFO] -------------------------------------------------------------

[ERROR] COMPILATION ERROR :

[INFO] -------------------------------------------------------------

[ERROR] /home/travis/build/DSH105/Nexus/src/main/java/com/dsh105/nexus/command/module/admin/TalkCommand.java:[38,27] error: cannot find symbol

[INFO] 1 error
```

**Comment by DSH105** [comment]
> Should this be free to use?

@DSH105    inb4 endless looping of talk command

File: src/main/java/com/dsh105/nexus/command/module/admin/TalkCommand.java (hunk 1 above)
Comment (lol768) [inline src/main/java/com/dsh105/nexus/command/module/admin/TalkCommand.java:39]: +1

**Pull request merged**